presto-cache/src/main/java/com/facebook/presto/cache/filemerge/CacheResult.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.cache.filemerge;
+
+public enum CacheResult
+{
+    /**
+     * The data we're reading is in cache
+     */
+    IN_CACHE,
+    /**
+     * The data we're reading is not in cache and we have quota to read them to cache
```

Comment:
nit: Did you mean to say "write them to cache"?

Reply:
I would prefer these to be standard names, HIT, MISS, etc.